docs/Troubleshooting.md

```diff
@@ -129,8 +129,7 @@ can debug your Jest tests with the following configuration:
       "type": "node",
       "request": "launch",
       "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/react-scripts",
-      "runtimeArgs": ["--inspect-brk", "test"],
-      "args": ["--runInBand", "--no-cache", "--env=jsdom"],
+      "args": ["test", "--runInBand", "--no-cache", "--env=jsdom"],
```

Comment:
Should `test` be there?

Reply:
As far as I know yes, as that's the exact example I got from VS Code's reply on the issue : https://github.com/Microsoft/vscode/issues/40896